### PR TITLE
Colinluthier test script change

### PR DESCRIPTION
**Pull request by colinluthier**
This adds the start time to the output dialog and adds .sh to the waterfall output dialog.

### CODE REVIEW

File: scripts/testing/start_scanning.sh [hunk 1]
```diff
@@ -21,15 +21,21 @@ scriptpath="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 # Change frequency here if you want to scan more/less.
 range="137M:138M:1k" # Format: StartFreq:StopFreq:Resolution
 
+echo "Start scanning at:"
+date
+
+
 echo "
+$(tput setaf 2)
```
Comment (jekhokie) [inline scripts/testing/start_scanning.sh:29]: Is this needed/is it duplicated from line 31?

**Review by jekhokie — CHANGES_REQUESTED** [review]

File: scripts/testing/start_scanning.sh [hunk 2]
```diff
@@ -21,15 +21,21 @@ scriptpath="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
 # Change frequency here if you want to scan more/less.
 range="137M:138M:1k" # Format: StartFreq:StopFreq:Resolution
 
+echo "Start scanning at:"
```
Comment (Cirromulus) [inline scripts/testing/start_scanning.sh:24]: Could you combine both echo blocks?

**Comment by colinluthier** [comment]
Currently the output looks like this:

```        Scanning for 5h, expect a timed_scan_06.03.21-11.50.csv.gz.png in /home/pi/raspberry-noaa-v2/scripts/testing afterwards.



        Scanning in range 137M:138M:1k, output file is timed_scan_06.03.21-11.50.csv.gz.

        Please note that this script will take precedence over all scheduled capture jobs!

        To stop scanning, either type `killall rtl_power`
        or use the script `/home/pi/raspberry-noaa-v2/scripts/testing/stop_and_finalize_scanning.sh timed_scan_06.03.21-11.50.csv.gz`.
        You can 'peek' while scanning with `/home/pi/raspberry-noaa-v2/scripts/testing/generate_waterfall timed_scan_06.03.21-11.50.csv.gz`.
        (note that gzip output is buffered in chunks, so it won't be updated immediately)
```
Having the date commmand would give the start time in local time as a convenience so it is easier to know when to chack back for results.

**Comment by jekhokie** [comment]
Closing as this was addressed as part of #250.

**Pull request closed without merging**